CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 2.0.3 (Next)
 
+* Add editorconfig - [@acm19](https://github.com/acm19).
```

Comment:
Let's put the PR number in the CHANGELOG like for the ones I've been making so it's clickable? I usually `git commit --amend` and `git push -f` after opening the PR.

Reply:
Also let's standardize on "Add" vs. "Added", "Fix" vs. "Fixed"? Maybe fix(ed) the remainder of CHANGELOG too?

Reply:
About your first comment done. About the second, I've created a ticket: https://github.com/acm19/aws-request-signing-apache-interceptor/issues/47, we can discuss it there, I'd prefer imperative and consistency between changelogs and commit headers. I've also created this one as a follow up: https://github.com/acm19/aws-request-signing-apache-interceptor/issues/48, I don't think we need manual toil for the CHANGELOG. Also, because of the size of this project I think we should do code review for **everything** but not necessarily create ticket for refactoring and internal development changes that don't affect lib users, I know it might bring confusion, but I wouldn't us to go through bureaucracy in a one class lib. Please let me know your thoughts.